### PR TITLE
Add ability to reveal password when signing up

### PR DESCRIPTION
**Pull request by jedireza**
Adds the ability to reveal the password when creating a new account.

### How was this tested?
- [x] I tested this manually

### Manual test cases
I tested in the simulator.

## Issues addressed
🔒 https://app.clickup.com/t/270u299

### CODE REVIEW

File: Client/Frontend/Intro/OtherOptionsPage.swift [hunk 1]
```diff
@@ -67,19 +68,38 @@ public struct EmailForm: View {
...
-                    SecureField("Password (required)", text: $password)
+                    ZStack(alignment: .trailing) {
+                        Group {
+                            if showPassword == true {
+                                TextField("Password (required)", text: $password)
+                            } else {
+                                SecureField("Password (required)", text: $password)
+                            }
+                        }
+                        .frame(height: 24.666667)
```
Comment (jedireza) [inline Client/Frontend/Intro/OtherOptionsPage.swift:79]: I don't love this, but it was the only way I found to keep the height of the `TextField` and `SecureField` the same. I derived that number by using a `GeometryReader` and showing the default height of each of these fields.

A text field on the left and secure field on the right:
image

Comment (jedireza) [reply Client/Frontend/Intro/OtherOptionsPage.swift:80]: For sure. I just added one.

**Review by chung-lai — APPROVED** [review]
LGTM. Would just suggest to double check this on other devices (smaller screen and iPad)

**Comment by jedireza** [comment]
Thanks @chung-lai. I ran the simulator for both iPhone Mini and iPad.

| iPhone Mini | iPad 11 in. |
| --- | --- |
| Simulator Screen Shot - iPhone 13 mini - 2022-04-27 at 17 09 38 | Simulator Screen Shot - iPad Pro (11-inch) (3rd generation) - 2022-04-27 at 17 10 58 |

**Pull request merged**